File: app foodPa/src/app/model/produto.ts
export class Produto {
    id: number;
    nome: string;
    valor: number;
    descricao: string;
    idRestaurante: number;

    constructor(){
        this.id =  0;
        this.nome = "";
        this.valor =  0;
        this.descricao = "";
        this.idRestaurante = 0;
    }
}

File: api foodPa/foodpa/foodpa/src/main/java/br/projeto/foodPa/model/Produto.java
package br.projeto.foodPa.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 *
 * @author dougl
 */
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Produto {
    private int id;
    private String nome;
    private String descricao;
    private String foto;
    private int valor;
    private int idRestaurante;
   
}
